src/Microsoft.DotNet.XHarness.Apple/AppTester.cs

```diff
@@ -428,17 +435,17 @@ public class AppTester : AppRunnerBase
 
                 envVariables[EnviromentVariables.HostName] = "127.0.0.1";
 
+                AddExtraEnvVars(envVariables, _appArguments);
+
                 var arguments = new List<string>
```

Comment:
It seems that `arguments` is unused variable

Reply:
Nice catch!